components/server/src/user/user-service.ts

```diff
@@ -330,11 +330,16 @@ export class UserService {
         return await this.userDb.storeUser(target);
     }
 
-    async findUserForLogin(params: { candidate: Identity }) {
+    async findUserForLogin(params: { candidate: IdentityLookup }) {
         let user = await this.userDb.findUserByIdentity(params.candidate);
         return user;
     }
 
+    async findOrganizationalUser(params: { organizationId: string; email: string }): Promise<MaybeUser> {
+        let user = await this.userDb.findOrganizationalUser(params.organizationId, params.email);
```

Comment:
Do we want some log-fields on this to be able to trace that later?

Reply:
done